crates/re_viewer_context/src/space_view/dyn_space_view_class.rs

```diff
@@ -44,12 +44,14 @@ pub enum SpaceViewClassLayoutPriority {
 pub trait DynSpaceViewClass {
     /// Name of this space view class.
     ///
-    /// Used for both ui display and identification.
-    /// Must be unique within a viewer session.
-    ///
-    /// TODO(#2336): Display name and identifier should be separate.
+    /// Used for identification. Must be unique within a viewer session.
     fn name(&self) -> SpaceViewClassName;
```

Comment:
the logical next/additional step would be to rename this to `identifier(&self) -> SpaceViewClassIdentifier` I think in order to avoid confusion

Reply:
I agree with that, but I'll punt on it as consistency would require a large sweep across the code base.

Reply:
I think it is worth doing though; do it in a follow-up PR if you like. rust-analyzers rename feature should make it pretty painless.